packages/api/helper/nominationGsheetToDB.js

```diff
@@ -72,18 +75,22 @@ module.exports = function gsheetToDB() {
               parseFloat(nomination[37].replace(/\$/g, '')) * 100
             ),
           },
-        }).then((array) => {
-          //db.Nomination.findOrCreate returns a promise that when fulfilled returns an array with two elements 
-          //the first element is the nomination instance object that contains the dataValues among other things  
-          //the second element is a boolean that represents whether a nomination was created
-          if (array[1]) {
+        })
+        if(array[1]){
```

Comment:
Can you make sure you have Prettier format on save enabled? If you're not sure how to do that, it's described here: https://www.digitalocean.com/community/tutorials/code-formatting-with-prettier-in-visual-studio-code

Reply:
Thanks, I have it installed but I didn't have it setup it auto save.  Corrected the issue!